### PR TITLE
Add  `p2p-direct-peers` CLI option

### PR DESCRIPTION
**Pull request by thedevbirb**
This PR updates the reference section of the docs for the CLI, describing the new `p2p-direct-peers` option introduced in Teku with https://github.com/Consensys/teku/pull/8098.

Thank you!

### CODE REVIEW

File: docs/reference/cli/index.md [hunk 1]
```diff
@@ -2178,10 +2178,50 @@ p2p-static-peers: ["/ip4/151.150.191.80/tcp/9000/p2p/16Ui...aXRz",
                     "/ip4/151.150.191.80/tcp/9000/p2p/16Ui...q6f1"]
 ```
 
+List of comma-separated [multiaddresses](https://docs.libp2p.io/concepts/appendix/glossary/#multiaddr) of static peers
+with which to establish and maintain connections.
+
+  </TabItem>
+</Tabs>
+
+### `p2p-direct-peers`
```
Comment (StefanBratanov) [inline docs/reference/cli/index.md:2187]: We keep an alphabetical order in the docs, so this should go above `p2p-discovery-bootnodes`

File: docs/reference/cli/index.md [hunk 2]
```diff
@@ -2178,10 +2178,50 @@ p2p-static-peers: ["/ip4/151.150.191.80/tcp/9000/p2p/16Ui...aXRz",
                     "/ip4/151.150.191.80/tcp/9000/p2p/16Ui...q6f1"]
 ```
 
+List of comma-separated [multiaddresses](https://docs.libp2p.io/concepts/appendix/glossary/#multiaddr) of static peers
+with which to establish and maintain connections.
+
+  </TabItem>
```
Comment (StefanBratanov) [inline docs/reference/cli/index.md:2184]:
```
  </TabItem>
</Tabs>
```

Should be above "List of comma-separated..."

**Review by StefanBratanov — COMMENTED** [review]
Thanks for the PR here as well! Everything look good, just comments about order and one wrongly placed tag

**Comment by thedevbirb** [comment]
@StefanBratanov sorry for the hiccups! I hope it is good now!

I've seen my editor has formatted a table also, if it is a problem I'll restore the change.

Thanks!

**Comment by StefanBratanov** [comment]
> @StefanBratanov sorry for the hiccups! I hope it is good now!
> 
> I've seen my editor has formatted a table also, if it is a problem I'll restore the change.
> 
> Thanks!

No worries at all. Formatting the table is ok given the linting checks pass. Will merge now. Thanks again!

**Review by StefanBratanov — APPROVED** [review]

**Pull request merged**